src/server/serverSetup.js

```diff
@@ -164,11 +169,16 @@ fluid.defaults("sjrk.storyTelling.server.app.storyTellingHandlers", {
             "route": "/stories/:id",
             "method": "get"
         },
-        saveStoryWithBinariesHandler: {
-            type: "sjrk.storyTelling.server.saveStoryWithBinariesHandler",
+        saveStoryHandler: {
+            type: "sjrk.storyTelling.server.saveStoryHandler",
             "route": "/stories/",
             "method": "post"
         },
+        saveStoryFileHandler: {
+            type: "sjrk.storyTelling.server.saveStoryFileHandler",
+            "route": "/stories/:id",
+            "method": "post"
+        },
```

Comment:
Should review the end points for proper RESTful semantics. In this case POST to create a story makes sense. However uploading a file would likely be better to POST to /stories/:id/files. Updating specific files, or stories can use PUT and deleting a file or story can use DELETE.

Reply:
Would also be good to have all of the story related files, that aren't in the database, stored in a directory named off the story id. This would make organization easier as well as deleting the story.

Reply:
As we discussed, the work to make the server API more closely follow RESTful semantics is out of scope for this Jira, so I've filed [SJRK-416](https://issues.fluidproject.org/browse/SJRK-416) to tackle this work.

The work to save files in directories named after the storyID rather than in the same directory with a filename based on the storyID still makes sense within the scope of SJRK-289.